gapic-generator/lib/gapic/presenters/service_rest_presenter.rb

```diff
@@ -127,6 +127,30 @@ def test_client_file_path
         main_service.service_file_path.sub ".rb", "_test.rb"
       end
 
+      ##
+      # @return [String]
+      #
+      def credentials_class_xref
```

Comment:
Is this used anywhere?

Reply:
Yes, it's used in the cloud's authentication.erb template.